Uchu.World/Systems/Behaviors/VentureVisionBehavior.cs

```diff
@@ -0,0 +1,41 @@
+using System.Threading.Tasks;
+using RakDotNet.IO;
+
+namespace Uchu.World.Systems.Behaviors
+{
+    public class VentureVisionBehavior : BehaviorBase
+    {
+        public override BehaviorTemplateId Id => BehaviorTemplateId.VentureVision;
+        private bool ShowCollectibles { get; set; }
+        private bool ShowMiniBosses { get; set; }
+        private bool ShowPetDigs { get; set; }
+        //what is the bitstream serialization for this?
+        //if i were to guess, it's the three bools + a duration, maybe just the duration, but i haven't tried any of that yet
```

Comment:
Any idea if there is a way to confirm all of this and actually implement it? Does DLU have this?

Reply:
DLU does not have it, you will need to actually work it out without stealing it from another server